client/rest/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -50,6 +49,14 @@ public Request(String method, String endpoint) {
         this.endpoint = Objects.requireNonNull(endpoint, "endpoint cannot be null");
     }
 
+    /**
+     * Copy the parameters from {@literal RequestConverters.Params} into {@linkplain Request}.
```

Comment:
I think that we should just say "Set the parameters" , who sets them may change in the future

Reply:
Also, note that RequestConverters are part of the high-level REST client while Request is part of the low-level REST client which is used independently.